std/StandardToken.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 contract StandardToken is Token {
-	uint256 public totalSupply;
```

Comment:
No, the change was fine. Please try the code in the compiler and see whether it produces bytecode for `B` or not.